README.md

```diff
@@ -4,3 +4,218 @@
 about formal systems. It features direct support for object-level
 binding constructs using higher order abstract syntax and treats
 contexts as first class objects.
+
+
+## Installation & Configuration Guide
```

Comment:
I remember @tsani said (I think similarly too) this Installation guide is too fragile to library change or so.
How about replacing this with the installation guide in readthedocs?

Reply:
I think we could have a small, summarized section for Install here at README; I'll send an update on that

Reply:
Yeah, I agree with that.

Reply:
Done